queries/bash/highlights.scm

```diff
@@ -62,6 +66,8 @@
  ("$" (special_variable_name))
  ] @constant
 
+(word) @string
```

Comment:
We don't have an `identifier` group https://github.com/nvim-treesitter/nvim-treesitter/blob/master/lua/nvim-treesitter/highlight.lua

Reply:
Then `@variable`?

Reply:
`FOO=bar` here bar is `word`, but bar is just a string, isn't a variable.

Reply:
`bar` being seen as a word instead of a string is something that should be fixed in the parser IMO. Maybe we should create an issue there?

Reply:
I feel a bit unsure with all `word` as string. The whole language sending strings to programs (=CLI arguments). Shouldn't there be a visual distinction whether something is quoted or not (and maybe even single quote since this causes a different behavior).

Reply:
maybe in this case, just check how `atom` does, which might resolve this issue. No one completely agrees here.

Reply:
I guess this is it https://github.com/atom/language-shellscript/blob/master/grammars/tree-sitter-bash.cson

I'm not sure what this means

https://github.com/atom/language-shellscript/blob/cad2413dc1b0b61f75f49dfae9e52a39e519f43d/grammars/tree-sitter-bash.cson#L105

Looks like they don't highlight `word` other than that

Reply:
yeah they seem to leave it unhighlighted. Which might be the best thing for now since the larser is so ambiguous